wemake_python_styleguide/visitors/ast/attributes.py

```diff
@@ -24,6 +30,19 @@ class WrongAttributeVisitor(BaseNodeVisitor):
         'mcs',
     ))
 
+    _func_stack: List[str] = []
```

Comment:
Any reason to use a class var?

Reply:
I think we can get context nodes and see if we are in a magic method that is inside a class.

Reply:
Interesting, I will try reworking using that.